src/containers/Registration.tsx

```diff
@@ -0,0 +1,209 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import React, { useState } from "react";
+import { Modal, Button, Form, FloatingLabel } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.css";
+import { useForm } from "react-hook-form";
+import styled from "styled-components";
+import * as yup from "yup";
+
+const WrapperDiv = styled.div`
+  > * {
+    font-family: TwitterChirp, -apple-system, BlinkMacSystemFont, "Segoe UI",
+      Roboto, Helvetica, Arial, sans-serif;
+  }
+`;
```

Comment:
thanks, but let's drop it. It'll be added in another place

Reply:
Deleted